qubes/mgmt.py

```diff
@@ -534,3 +570,95 @@ def vm_kill(self):
         assert not self.arg
         self.fire_event_for_permission()
         yield from self.dest.kill()
+
+    @api('mgmt.Events', no_payload=True)
```

Comment:
Should it be that way? Or don't we want to write some other, optimised socket interface especially for this function? I don't know. This class (and approach) seems just barely adequate...